stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/ChannelsView.kt

```diff
@@ -31,14 +34,25 @@ public class ChannelsView @JvmOverloads constructor(
 
     init {
         addView(channelListView, LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.MATCH_PARENT))
+
         emptyStateView.apply {
             isVisible = false
             addView(this, defaultChildLayoutParams)
         }
+
         loadingView.apply {
             isVisible = false
             addView(loadingView, defaultChildLayoutParams)
         }
+
+        setMoreOptionsClickListener { channel ->
+            context.getFragmentManager()?.let { fragmentManager ->
+                ChannelActionsDialogFragment
+                    .newInstance(channel.cid, !channel.isDirectMessaging())
+                    .show(fragmentManager, null)
+            }
```

Comment:
Could we close the swiped item when we open this dialog?

Reply:
I'll check with design and see what they want.